src/PhpSpec/Formatter/Presenter/StringPresenter.php

```diff
@@ -359,4 +366,75 @@ private function presentCallable($value)
 
         return sprintf('[%s()]', $value);
     }
+
+    /**
+     * @param UnexpectedCallException $exception
+     *
+     * @return string
+     */
+    private function presentCallArgumentsDifference(UnexpectedCallException $exception)
+    {
+        $text = '';
+
+        $actualArguments = $exception->getArguments();
+        $methodProphecies = $exception->getObjectProphecy()->getMethodProphecies($exception->getMethodName());
+        $presentedMethodProphecy = $this->findMethodProphecyOfFirstNotExpectedCall($methodProphecies, $exception);
+        $expectedTokens = $presentedMethodProphecy->getArgumentsWildcard()->getTokens();
+
+        if (count($expectedTokens) !== count($actualArguments)) {
+
+            //parameters count mismatch will be shown in exception message, no need in diff
+            return $text;
+        }
+
+        $expectedArguments = array();
+        foreach ($expectedTokens as $token) {
+            if ($token instanceof ExactValueToken) {
+                $expectedArguments[] = $token->getValue();
+            } else {
+                $expectedArguments[] = (string)$token;
+            }
+        }
+
+        foreach($actualArguments as $i => $actualArgument) {
+            $expectedArgument = $expectedArguments[$i];
+            if (is_null($actualArgument)) {
+                $actualArgument = 'null';
+            }
+            if (is_null($expectedArgument)) {
```

Comment:
All arguments that are equal to false (empty string/false/numeric zero) will be outputed as 'null' if I do so. I will use ternary operator instead of `if` here:
```php
$actualArgument = is_null($actualArgument) ? 'null' : $actualArgument;
```